docs/operator/event_type.md

```diff
@@ -0,0 +1,22 @@
+---
+id: event_type
+title: Event Types
+---
+
+## Event Types
```

Comment:
Also here, call it notifications

Reply:
Solved on https://github.com/rucio/documentation/pull/233/commits/ba3ca96909cdd479a1ca218826c0d8e15436a6b5